tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/OrdersControllerTests.cs

```diff
@@ -633,15 +614,12 @@ public async Task UpdateStatusAsync_OrderIsComplete_LastUpdatedIsUpdated()
 
             await context.OrdersController.UpdateStatusAsync("Order", context.CompleteOrderStatusModel);
 
-            context.Order.LastUpdatedBy.Should().Be(context.NameIdentity);
-            context.Order.LastUpdatedByName.Should().Be(context.Name);
-            context.Order.LastUpdated.Should().NotBe(DateTime.MinValue);
+            context.CompleteOrderServiceMock
+                .Verify(x => x.CompleteAsync(It.IsNotNull<CompleteOrderRequest>()), Times.Once);
```

Comment:
Could update this to instead of doing It.IsNotNull, use It.Is<CompleteOrderRequest>(req => req.Order == context.Order), I think, which would be a bit more specific